www/docs/customization/aur.md

```diff
@@ -163,8 +162,8 @@ aurs:
 ```
 
 !!! tip
-    Learn more about the [name template engine](/customization/templates/).
+Learn more about the [name template engine](/customization/templates/).
 
 !!! tip
-    For more info about what each field does, please refer to
-    [Arch's PKGBUILD reference](https://wiki.archlinux.org/title/PKGBUILD).
+For more info about what each field does, please refer to
+[Arch's PKGBUILD reference](https://wiki.archlinux.org/title/PKGBUILD).
```

Comment:
this will break mkdocs afaik

Reply:
several other places too, whenever there's a `!!! someword` and then text is indented the next line

see: https://squidfunk.github.io/mkdocs-material/reference/admonitions/

Reply:
Forgot I left format-on save on, let me redo and reapply!